src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1738,9 +1739,22 @@ public bool IsPassive()
             return _engine.CurrentState == RachisState.Passive;
         }
 
-        public Task<(long Index, object Result)> SendToLeaderAsync(CommandBase cmd)
+        public async Task<(long Index, object Result)> SendToLeaderAsync(CommandBase cmd)
         {
-            return SendToLeaderAsyncInternal(cmd);
+            var response = await SendToLeaderAsyncInternal(cmd);
+
+            #if DEBUG
+            
+            //this is a temporary measure until RavenDB-11761 is done
```

Comment:
Will probably keep this in place to make sure that we aren't making this mistake, though.